### DISCUSSION TITLE
'Sync' icon disappears from ribbon

### DISCUSSION THREAD

**Discussion by LemurTech** [thesamim/TickTickSync#147] [General]
What is the mechanism whereby the 'Sync' icon gets into the left-hand ribbon? Mine seems to disappear on occasion. It's not 'hidden', it's just gone from the configuration menu options, and I can't figure out how to reset it. Reinstalling TickTickSync doesn't seem to do the trick.

**Comment by thesamim** [comment]
@LemurTech The sync Icon only shows up in the ribbon when "Debug Mode" is turned on. 

If "Debug Mode" is turned on, and you're not seeing the Icon, please let me know.

**Reply by LemurTech** [reply]
Gosh, I'm a dummy. I could have figured that out! Thank you!

**Comment by thesamim** [comment]
@LemurTech : FYI: The next release the Sync Button will always be in the ribbon....

**Comment by thesamim** [comment]
as of 1.0.34 ribbon button is always on.
